fnv.go

```diff
@@ -1,5 +1,8 @@
 package bigcache
 
+import(
+	"github.com/cespare/xxhash"
```

Comment:
Can you please provide a justification on why you want to use a 3rd party hashing algorithm?